### DISCUSSION TITLE
webpack loader?

### DISCUSSION THREAD

**Discussion by devmattrick** [vanilla-extract-css/vanilla-extract#282] [Q&A] [answered]
Previously: #213 

I'm in the same boat as @jimkyndemeyer where I'd like to use TypeScript without requiring Babel. I think a simple way to achieve this would be to make a webpack loader for vanilla-extract that allows for users to chain loaders like:
```js
{
    test: /\.css\.ts$/,
    use: [
        "@vanilla-extract/loader",
        "ts-loader"
    ]
},
```

I'd love to hear your thoughts on this.

**Comment by mattcompiles** [comment]
This is definitely something we're considering doing to align webpack with the other integrations.

**Comment by mattcompiles** [comment, marked as answer]
This should be resolved as of `@vanilla-extract/webpack-plugin@2.1.0`.
